tests/test_usm_utils.py

```diff
@@ -0,0 +1,42 @@
+import dpctl.tensor as dpt
+from numpy.testing import assert_equal
+
+import dpnp
+
+
+def test_byte_bounds():
+    a = dpt.usm_ndarray((3, 4), dtype="int64", buffer="shared")
+    values = dpt.arange(12, dtype="int64")

```

Comment:
Our main use case is to test `dpnp.ndarray` interface rather than `dpt.usm_ndarray`:
```suggestion
    a = dpnp.zeros((3, 4), dtype=dpnp.int64, usm_type="shared")
    values = dpnp.arange(12, dtype=dpnp.int64)

```